.github/workflows/create_release_assets.yml

```
@@ -26,6 +26,8 @@ jobs:
         # https://github.com/topgrade-rs/topgrade/issues/1095
         platform: [ ubuntu-22.04, macos-latest, macos-15-intel, windows-latest ]
     runs-on: ${{ matrix.platform }}
+    env:
+      event_tag: ${{ github.event.client_payload.tag }}
```

Here and all other `env: event_tag: ...` usages. Just a bit easier to read, unless you object?
```suggestion
      tag: ${{ github.event.client_payload.tag }}
```